content/common/linux.mk

```
@@ -0,0 +1,12 @@
+CC := gcc
```

```suggestion
CC = gcc
```
No need to use `:=`.